source/whats-new.txt

```diff
@@ -56,6 +57,44 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.5:
+
+What's New in 6.5
+-----------------
+
+The {+driver-short+} v6.5 release includes the following features:
+
+- Moved ``ObjectId`` generation to the ``pkFactory`` class for bulk write
+  operations.
+
+.. warning::
```

Comment:
Q: should this warning be indented to be under the preceding bullet point?

Reply:
I'm not sure you can put an admonition in a list, but I'll try it out.

Reply:
Unfortunately, that didn't work (unexpected indentation error)